src/test/java/io/aiven/kafka/connect/gcs/TopicPartitionRecordGrouperTest.java

```diff
@@ -73,41 +75,24 @@
     private static final SinkRecord T1P1R3 = new SinkRecord(
         "topic1", 0, Schema.OPTIONAL_STRING_SCHEMA, "some_key", null, null, 1003);
 
-    @ParameterizedTest
-    @ValueSource(strings = {
-        "",
-        "{{topic}}", "{{partition}}", "{{start_offset}}",
-        "{{topic}}-{{partition}}", "{{topic}}-{{start_offset}}", "{{partition}}-{{start_offset}}",
-        "{{topic}}-{{partition}}-{{start_offset}}-{{unknown}}"
-    })
-    final void incorrectFilenameTemplates(final String template) {
-        final Template filenameTemplate = Template.of(template);
-        final IllegalArgumentException ex = assertThrows(IllegalArgumentException.class,
-            () -> new TopicPartitionRecordGrouper(filenameTemplate, null));
-        assertFalse(TopicPartitionRecordGrouper.acceptsTemplate(filenameTemplate));
-        assertEquals(
-            "filenameTemplate must have set of variables {topic,partition,start_offset}, but {"
-                + String.join(",", filenameTemplate.variables())
-                + "} was given",
-            ex.getMessage());
-    }
```

Comment:
No value in these tests (and also in `KeyRecordGrouperTest.java`)?

Reply:
it is valuable and has been moved to the `GcsSinkConfigTest`

Reply:
Right, sorry!